python/custreamz/custreamz/kafka.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, kafka_configs):
 
         Parameters
         ----------
-        kafka_configs, dict, Key/Value pairs of librdkafka
+        kafka_configs : dict, Dict of Key/Value pairs of librdkafka
         configuration values. Full list of valid configuration
         options can be found at
         https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md
```

Comment:
This is still misformatted, first line is parameter name and type, second+ lines are indented and description of parameter:
```suggestion
        kafka_configs : dict,
            Dict of Key/Value pairs of librdkafka configuration values. Full
            list of valid configuration options can be found at
            https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md
```

Reply:
This needs to be fixed for all the docstrings

Reply:
Additionally, there's still docstrings with `Parameters None` and empty line breaks that should be fixed as well